core/common/src/main/resources/alluxio-default.properties

```diff
@@ -148,14 +148,14 @@ alluxio.user.failed.space.request.limits=3
 alluxio.user.file.buffer.bytes=1MB
 alluxio.user.file.master.client.threads=10
 alluxio.user.file.readtype.default=CACHE_PROMOTE
-alluxio.user.file.cache.partially.read.block=false
+alluxio.user.file.cache.partially.read.block=true
```

Comment:
I have changed this to true by default because I think it is annoying to ask spark users to set this to true everytime. Let me know what you think. thanks